### PR TITLE
Adds missing Barcelona stations

### PR DESCRIPTION
**Pull request by DCoventryTrainline**
Adds missing Barcelona stations.

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -5008,7 +5008,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6608;Andorre;andorre;;;;;;ES;Europe/Madrid;t;f;f;f;f;f;ESAND;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6608;
 6609;Arévalo;arevalo;7112105;;;;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;23715;;;;;;;;;;;アレバロ;;;;Аревало;;;阿雷瓦洛;urn:trainline:public:nloc:csv23715;
 6610;La Garrovilla;la-garrovilla;7137601;;38.915453;-6.474067;;ES;Europe/Madrid;f;f;f;t;f;f;ESARJ;;f;;f;;f;eygmkt;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;37601;t;;f;ESARJ;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6610;
-6611;Arbos;arbos;7172202;;;;;ES;Europe/Madrid;f;f;f;f;f;f;ESARS;;f;;f;7172202;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;ESARS;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6611;
+6611;L’arboc;larboc;7172202;;41.273412;1.600516;;ES;Europe/Madrid;f;f;f;f;f;f;ESARS;;f;;f;7172202;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;ESARS;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6611;
```
Comment (meganaslattery) [inline stations.csv:5011]: The station name is L'Arboç I believe.

Comment (DCoventryTrainline) [reply stations.csv:5011]: This should be fixed

File: stations.csv [hunk 2]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
 74453;Havneby;havneby;;;55.086166;8.569992;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8600921;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74453;
+74454;Garraf;garraf;7171703;;41.254712;1.904007;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74454;
+74455;Els Monjos;els-monjos;7172203;;41.316708;1.662007;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74455;
+74456;Lavern Subirats (Apd);lavern-subirats-apd;7172206;;41.395755;1.768162;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74456;
+74457;Sant Sadurni D’anoia;sant-sadurni-danoia;7172207;;41.419824;1.794186;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74457;
```
Comment (meganaslattery) [inline stations.csv:70316]: Looking on the Cercanias map this should be Sant Sadurni d’Anoia

File: stations.csv [hunk 3]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
 74452;Schlatt TG Bahnhof;schlatt-tg-bahnhof;;;47.679333;8.687562;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8573491;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74452;
 74453;Havneby;havneby;;;55.086166;8.569992;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8600921;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74453;
+74454;Garraf;garraf;7171703;;41.254712;1.904007;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74454;
+74455;Els Monjos;els-monjos;7172203;;41.316708;1.662007;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74455;
+74456;Lavern Subirats (Apd);lavern-subirats-apd;7172206;;41.395755;1.768162;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74456;
```
Comment (meganaslattery) [inline stations.csv:70315]: Lavern Subirats (Apd) - is this the station name?

File: stations.csv [hunk 4]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74496;Sant Adrià De Besòs;sant-adria-de-besos;7179403;;41.424665;2.230761;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74496;
+74497;Montgat Nord;montgat-nord;7179406;;41.468941;2.286615;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74497;
+74498;El Masnou;el-masnou;7179407;;41.477702;2.314362;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74498;
+74499;Ocata (Apd);ocata-apd;7179408;;41.4789679;2.3165779;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74499;
+74500;Premiá De Mar;premia-de-mar;7179409;;41.4875728;2.354368;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74500;
```
Comment (meganaslattery) [inline stations.csv:70359]: It's the wrong accent I think: `Premià de Mar`

Comment (DCoventryTrainline) [reply stations.csv:70359]: This should be fixed

File: stations.csv [hunk 5]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74487;Campdevànol;campdevanol;7177301;;42.221558;2.165877;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74487;
+74488;Planoles;planoles;7177304;;42.313559;2.103078;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74488;
+74489;Toses;toses;7177305;;42.324189;2.029728;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74489;
+74490;Urtx Alp;urtx-alp;7177307;;42.3761666;1.897009;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74490;
+74491;Montcada I Reixac;montcada-i-reixac;7179005;;41.483833;2.188193;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74491;
```
[the remark this answers is not in the document]
Comment (DCoventryTrainline) [reply stations.csv:70350]: This should be fixed

File: stations.csv (hunk 3 above)
Comment (DCoventryTrainline) [reply stations.csv:70315]: This should be fixed

File: stations.csv (hunk 2 above)
Comment (DCoventryTrainline) [reply stations.csv:70316]: This should be fixed

File: stations.csv [hunk 6]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74465;El Prat Aeroport;el-prat-aeroport;7172400;;41.303844;2.07218;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74465;
+74466;Rubí Can Vallhonrat;rubi-can-vallhonrat;7172501;;41.475318;2.037135;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74466;
+74467;Sant Cugat Coll Favà;sant-cugat-coll-fava;7172502;;41.481295;2.080439;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74467;
+74468;Cerdanyola-Universitat;cerdanyola-universitat;7172503;;41.496992;2.115035;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74468;
+74469;Santa Perpètua De Mogoda Riera de Caldes;santa-perpetua-de-mogoda-riera-de-caldes;7172508;;41.5246497;2.1877809;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74469;
```
Comment (meganaslattery) [inline stations.csv:70328]: Santa Perpètua de Mogoda Riera de Caldes

File: stations.csv [hunk 7]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74471;Santa Perpètua de Mogoda La Florida;santa-perpetua-de-mogoda-la-florida;7177003;;41.521426;2.193917;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74471;
+74472;Mollet Santa Rosa;mollet-santa-rosa;7177004;;41.535064;2.205846;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74472;
+74473;Parets del Vallès;parets-del-valles;7177005;;41.5561681;2.225913;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74473;
+74474;Granollers Canovelles;granollers-canovelles;7177006;;41.6113633;2.2769926;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74474;
+74475;Les Franqueses Del Vallès;les-franqueses-del-valles;7177100;;41.624639;2.298077;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74475;
```
Comment (meganaslattery) [inline stations.csv:70334]: Les Franqueses del Vallès

File: stations.csv [hunk 8]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74476;La Garriga;la-garriga;7177102;;41.684575;2.288761;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74476;
+74477;Figaró;figaro;7177103;;41.7227863;2.2704679;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74477;
+74478;Sant Martí De Centelles;sant-marti-de-centelles;7177104;;41.760542;2.24972;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74478;
```
Comment (meganaslattery) [inline stations.csv:70337]: Sant Martí De Centelles

File: stations.csv [hunk 9]
```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+74503;Sant Andreu de Llavaneres;sant-andreu-de-llavaneres;7179501;;41.555964;2.4992651;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74503;
+74504;Caldes d’Estrac;caldes-destrac;7179502;;41.56922;2.5254071;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74504;
+74505;Canet de Mar;canet-de-mar;7179601;;41.587123;2.583146;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74505;
+74506;Tordera;tordera;7179607;;41.587127;2.5805711;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74506;
+74507;Santa Susann;santa-susann;7179608;;41.629767;2.7114611;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74507;
```
Comment (MontseBadia) [inline stations.csv:70366]: Hello, I believe this station is called Santa Susanna

Comment (polmtn) [inline stations.csv:70366]: This should be Santa Susanna / santa-susanna. It's missing the 'a' at the end.

File: stations.csv [hunk 10]
```diff
@@ -5075,7 +5075,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6675;Pamplona/Iruña;pamplona-iruna;7180100;;42.825122;-1.661889;;ES;Europe/Madrid;f;f;f;t;f;f;ESPNA;;f;;f;7100080;f;ezwgdd;t;ESPNA;t;;f;;f;;f;;f;;f;;f;;;;f;;f;80100;t;;f;ESPNA;f;;f;f;;;;;Pamplune;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6675;
 6676;Port-Bou;port-bou;;;;;;ES;Europe/Madrid;f;f;f;f;f;f;ESPOB;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;4986;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4986;
 6677;Tarragona Estació;tarragona-estacio;7171500;;41.111212;1.253527;24444;ES;Europe/Madrid;f;t;f;t;f;t;ESQGN;;f;;f;7100062;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;71500;t;;f;ESQGN;f;;f;f;;;;;Tarragone;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6677;
-6678;Ribas de Freser;ribas-de-freser;7177303;;;;;ES;Europe/Madrid;f;f;f;f;f;f;ESRDF;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;ESRIB;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6678;
+6678;Ribas de Freser;ribas-de-freser;7177303;;42.301158;2.166078;;ES;Europe/Madrid;f;f;f;f;f;f;ESRDF;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;ESRIB;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6678;
```
Comment (MontseBadia) [inline stations.csv:5078]: The official catalan name for this one is Ribes de Freser

File: stations.csv (hunk 9 above)
Comment (DCoventryTrainline) [reply stations.csv:70366]: Thanks, That should be fixed

File: stations.csv (hunk 10 above)
Comment (DCoventryTrainline) [reply stations.csv:5078]: This should be fixed as well

File: stations.csv (hunk 7 above)
Comment (DCoventryTrainline) [reply stations.csv:70334]: This should be fixed

**Review by meganaslattery — APPROVED** [review]

File: stations.csv [hunk 11]
```diff
@@ -4948,13 +4948,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6548;Algeciras;algeciras;7155020;;36.12672;-5.450995;;ES;Europe/Madrid;f;f;f;t;f;f;ESAFR;;f;;f;7100136;f;eykjeb;t;ESAEI;t;23571;t;;f;;f;;f;;f;;f;;;;f;;f;55020;t;;f;ESAFR;f;;f;f;;;;;Algésiras;;;;;;;アルヘシラス;알헤시라스;;;Альхесирас;;;阿尔赫西拉斯;urn:trainline:public:nloc:csv6548;
 6549;Sagunt;sagunt;7165200;;39.675779;-0.271564;;ES;Europe/Madrid;f;f;f;t;f;f;ESAFS;;f;;f;7100055;f;ezpccb;t;;f;32391;t;;f;;f;;f;;f;;f;;;;f;;f;65200;t;;f;ESAFS;f;;f;f;;;;Sagunto;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6549;
 6550;Sabadell Sud;sabadell-sud;7178703;;41.528725;2.105183;27261;ES;Europe/Madrid;f;f;f;t;f;f;ESAFT;;f;;f;7178703;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;78703;t;;f;ESAFT;f;;f;f;;;;Sur;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6550;
-6551;Puigcerdà;puigcerda;7100291;;42.4316;1.92819;;ES;Europe/Madrid;f;f;f;t;f;f;ESAFU;;f;;f;;f;sp937u;t;ESPUI;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6551;
+6551;Puigcerdà;puigcerda;7100291;;42.4316;1.92819;;ES;Europe/Madrid;f;f;f;f;f;f;ESAFU;;f;;f;;f;sp937u;t;ESPUI;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6551;
```
Comment (meganaslattery) [inline stations.csv:4951]: Why have you changed is suggestible to false on this station?

File: stations.csv [hunk 12]
```diff
@@ -28703,7 +28703,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 30855;Rioseras;rioseras;;;42.4537965;-3.6398029;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;ezt3t0;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30855;
 30856;Riovena;riovena;;;43.2472396;-6.5832063;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;ezeq40;t;ESOYB;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30856;
 30857;Rípodas;ripodas;;;42.6864006;-1.3060756;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;ezx4f6;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30857;
-30858;Ripoll;ripoll;;;42.20064;2.19033;;ES;Europe/Madrid;t;f;f;t;t;f;;;f;;f;;f;sp981f;t;ESOEL;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30858;
+30858;Ripoll;ripoll;7177200;;42.195536;2.195977;;ES;Europe/Madrid;f;f;f;f;t;f;;;f;;f;;f;sp981f;t;ESOEL;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv30858;
```
Comment (meganaslattery) [inline stations.csv:28706]: I think that this is the city station. I'm not sure that we necessarily want to convert it to the Cercanias station?

Comment (meganaslattery) [reply stations.csv:28706]: I think the same comment applies to `Montgat`.

File: stations.csv (hunk 11 above)
Comment (DCoventryTrainline) [reply stations.csv]: Not sure how that happened but its fixed now

File: stations.csv (hunk 12 above)
Comment (DCoventryTrainline) [reply stations.csv:28706]: This should be fine, these locations currently don't do anything so it seems that we may as well convert them into something useful

File: stations.csv (hunk 11 above)
Comment (meganaslattery) [reply stations.csv]: I think the same comment applies to other stations - for example, `Ripoll`, `Pineda de Mar`, and `Santa Susanna`. Perhaps others also.

File: stations.csv (hunk 12 above)
Comment (meganaslattery) [reply stations.csv:28706]: I think that we are returning some journeys when searching the city station so we might want to consider that.

Comment (DCoventryTrainline) [reply stations.csv:28706]: This should be fixed now

Comment (meganaslattery) [reply stations.csv:28706]: I think that `is_suggestible` is false for this station - do we want to stop suggesting it?

**Pull request closed without merging**